pages/linux/zramctl.md

```diff
@@ -0,0 +1,23 @@
+# zramctl
+
+> Setup and control zram devices.
+
+- Check if zram is enabled:
+
+`lsmod | grep -i zram`
+
+- Enable zram with 2 devices:
```

Comment:
> And if so, where is the size specified ?

Simply creating the disk doesn't initialise it to have a size. You need to use `zramctl` to do that

I've written [a blog post](https://starbeamrainbowlabs.com/blog/article.php?article=posts%2F226-Linux-zram.html) on it, if it helps :-)